src/psij/web/js/customization.js

```diff
@@ -0,0 +1,57 @@
+CUSTOMIZATION = {
+    projectName: "PSI/J"
+}
+
+var CONF = {
+    STR: {
+        "dashboardTitle": "Exascale dashboard testing service",
+        "introBlurb": "The dashboard testing service shows you the status of various sites in the calendar view by date and machine without curation.",
+        "bottomBlurb": "The curated sites tab will eventually show labs on the left which are expandable to show the machines of each lab on the y axis.  Meanwhile, the X axis will show you test cases like PSI/J and PSI/J Parcel, etc."
+    },
+    "curated_sites": {
```

Comment:
Here's what we came up with at the product discussion meeting.  here's a quick recap:
Dan Laney and team came up with a project objective of creating a new tab called curated sites.  In curated sites we wanted to have a list of curated machines that we show to the user on it's own tab.  they were to be separated by accordians for each lab, so as to be grouped by each lab.  they wanted all this to be configurable from a config file so we could change the list of machines to be shown.

Reply:
I am not familiar with this, so it must be a meeting I missed. I remember a discussion in which we wanted to show logos for some of the sites, which would have to be input manually. Is this related?

I think it would be best if we opened an issue with a clear statement of the problem and work from there.